### PR TITLE
arm64: dts: rockchip: radxa e20c: add leds

### PR DESCRIPTION
**Pull request by RadxaSongjun**
arm64: dts: rockchip: radxa e20c: add sys/wan/lan leds

Signed-off-by: SongJun Li <lisongjun@radxa.com>

### CODE REVIEW

File: arch/arm64/boot/dts/rockchip/rk3528-radxa-e20c.dts [hunk 1]
```diff
@@ -155,6 +155,26 @@
...
+		pinctrl-names = "default";
+
+		sys-led {
+			gpios = <&gpio4 RK_PC1 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "heartbeat";
+		};
+
+		wan-led {
+			gpios = <&gpio4 RK_PC0 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "none";
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/rk3528-radxa-e20c.dts]: https://github.com/torvalds/linux/blob/master/arch/arm/boot/dts/intel/ixp/intel-ixp42x-dlink-dsm-g600.dts#L51

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/rockchip/rk3528-radxa-e20c.dts (hunk 1 above)
Comment (RadxaSongjun) [reply arch/arm64/boot/dts/rockchip/rk3528-radxa-e20c.dts]: 这个尝试过trigger为netdev 并且在sys写入到trigger了 但是灯没闪烁 Stephen说先基于这个提交

Comment (RadxaYuntian) [reply arch/arm64/boot/dts/rockchip/rk3528-radxa-e20c.dts]: 要在sysfs里面配置吧？
https://kernel.googlesource.com/pub/scm/linux/kernel/git/nvdimm/nvdimm/+/libnvdimm-for-next/Documentation/ABI/testing/sysfs-class-led-trigger-netdev
https://gist.github.com/FrankWu100/25779214ea0981f7d3095014a815a10f

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
squash提交

**Review by RadxaYuntian — APPROVED** [review]

**Comment by RadxaStephen** [comment]
Added it to branch linux-5.10-gen-rkr4.1.

**Pull request closed without merging**